perf/run-benchmarks

```diff
@@ -453,18 +579,21 @@ def run_benchmarks(
     corpuses = SMALL_CORPUSES + MEDIUM_CORPUSES
     if dummy:
         corpuses = DUMMY_CORPUSES
-    if internal:
+    elif internal:
         corpuses = INTERNAL_CORPUSES
-    if gitlab:
```

Comment:
I feel like there might be a case where you want to run internal and github corpuses? not super important right now tho 🤷

Reply:
Yeah, I didn't want to change any of the existing functionality for right now. If we decide to make things easier in the future, we can do that

Reply:
yep, sounds good 👍